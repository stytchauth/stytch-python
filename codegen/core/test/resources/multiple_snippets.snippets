snippet1=hello world
snippet2=goodbye
