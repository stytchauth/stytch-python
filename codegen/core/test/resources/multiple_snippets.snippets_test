value = 123
# MANUAL(snippet1)
old_snippet1
# ENDMANUAL(snippet1)
value2 = 456
# MANUAL(snippet2)
# ENDMANUAL(snippet2)
value3 = 3.14
