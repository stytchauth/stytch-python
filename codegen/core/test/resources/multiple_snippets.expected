value = 123
# MANUAL(snippet1)
hello world
# ENDMANUAL(snippet1)
value2 = 456
# MANUAL(snippet2)
goodbye
# ENDMANUAL(snippet2)
value3 = 3.14
